app/openOrders.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { FlatList, Text, View } from 'react-native'
+import { List } from 'react-native-paper'
+import Ionicons from '@expo/vector-icons/Ionicons'
+
+import Page from '@components/Page'
+import { selectProfileData } from '@store'
+
+const OpenOrders = () => {
+    const profileData = useSelector(selectProfileData)
+
+    const orders = profileData?.openOrders
+
+    const orderList = (
+        <FlatList
```

Comment:
eventually this can be our custom component

Reply:
Agreed. That's a change I'll make to the buy branch once I merge these changes.